### ISSUE TITLE
Any chance for GGPO support?

### ISSUE THREAD

**Issue by malvarenga123** [finalburnneo/FBNeo#700]
Hi there, I know there was a licensing issue with the GGPO client in the past.
I was wondering if this was still the case, considering both the code for the SDK and for Fightcade (which uses FBNeo) are available here:
https://github.com/JordiRos/fightcade-fbneo
https://github.com/pond3r/ggpo

**Comment by barbudreadmon** [comment]
I'm not a license expert, but i don't think MIT license is compatible with FBNeo license, it looks like a GPL license that doesn't enforce to share your changes, GPL being already incompatible. In the first place, thinking that it's legal because the fightcade guys are doing it would be a big mistake, since they are already breaking several points of [our license](https://github.com/finalburnneo/FBNeo/blob/master/src/license.txt).

**Comment by malvarenga123** [comment]
Just out of curiosity, what exactly is the FBNeo license? Wouldn't it be better/easier to just use the GPL or something similar?

**Comment by barbudreadmon** [comment]
I gave the link to our license above.

As for changing license : 
- changing license requires authorization from all people that ever contributed to our project, or to remove their code, that's not gonna happen
- we want FBNeo to stay non-commercial anyway

**Comment by malvarenga123** [comment]
I saw the license, that's why I asked, because it doesn't look like GPL, BSD, Apache, etc.

**Comment by barbudreadmon** [comment]
Yeah, it's what you would call a custom license, because anyone can create its own license, they are just a bunch of rules you apply to the usage of your source code.

**Comment by barbudreadmon** [comment]
i did some search, MIT doesn't appear to be a copyleft license, so i think there is no compatibility issue.
